server/src/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -968,7 +968,11 @@
         }
 
         if (state != null) {
-            sc.setParameters("stateEQ", state);
+            if (state.equalsIgnoreCase("present")) {
```

Comment:
Good catch, IMHO `Expunged` is not necessary but `Expunging` like we use for user e.g. https://github.com/resmo/cloudstack/blob/fix/CLOUDSTACK-8578-extend/server/src/com/cloud/api/query/QueryManagerImpl.java#L984